etsin_finder/frontend/js/components/qvain/files/response.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Translate from 'react-translate-component'
+import Loader from '../../general/loader'
+
+// Shows success/fail based on a RPC request response. If the response prop is null, shows a loader.
+class Response extends Component {
```

Comment:
I would recommend using stateless functional components as much as possible instead of stateful class components (reasoning -> [Functional-vs-class-components-in-react](https://medium.com/@Zwenza/functional-vs-class-components-in-react-231e3fbd7108) )

Reply:
good idea, done